spec/models/user_spec.rb

```diff
@@ -10,6 +10,8 @@
 
   it { is_expected.to accept_nested_attributes_for :status }
 
+  it { is_expected.to respond_to :status_count }
```

Comment:
I think it would be better to test what `status_count` is doing rather than just testing that it responds to the method